packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -80,6 +84,17 @@ export async function activate(context: vscode.ExtensionContext) {
       if (languageClient) {
         languageClient.onNotification('indexer/done', async () => {
           LanguageClientUtils.indexing = false;
+
+          // Refresh SObject definitions if there aren't any faux classes
+          const sobjectRefreshStartup: boolean = vscode.workspace
+            .getConfiguration(SFDX_APEX_CONFIGURATION_NAME)
+            .get<boolean>(ENABLE_SOBJECT_REFRESH_ON_STARTUP, true);
```

Comment:
The default should be `false`

Reply:
I'll change that for consistency. FYI I recently discovered this value doesn't actually matter here, the default value is read from the package.json.